src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestNegative.cs

```diff
@@ -42,6 +42,7 @@ private static string GetResourceStringFromReflection(string key)
         /// For NuGet EventSources we validate both "runtime" and "validation" behavior
         /// </summary>
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Full Framework doesn't have System.SR type which is used in this test.")]
```

Comment:
> Full Framework doesn't have System.SR type which is used in this test

The test uses the above GetResourceString helper to fish the exception message out of resources.  On core that's done via reflection to the SR property.  Is there are a way to do it on desktop such that rather than disabling the test, we could branch in GetResourceString to do something different for desktop to get the string another way?